### PR TITLE
Build genutil 8.2.1

### PR DESCRIPTION
**Pull request by muryanto1**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,12 +1 @@
...
-export LDFLAGS="-L${PREFIX}/lib"
-
-if [[ `uname` == "Linux" ]]; then
-    export LDSHARED_FLAGS="-shared -pthread"
-else
-    export LDSHARED_FLAGS="-bundle -undefined dynamic_lookup"
-fi
-export LDSHARED="$CC $LDSHARED_FLAGS"
-LDSHARED=$LDSHARED $PYTHON -m pip install . --no-deps -vv
+$PYTHON -m pip install . --no-deps -vv
```
Comment (xylar) [inline recipe/build.sh]: Might even be worth getting rid of `build.sh` if it's only one line (in a future PR).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,10 +7,10 @@ package:
 
 source:
     url: https://github.com/cdat/genutil/archive/v{{ version }}.tar.gz
-    sha256: 171dd39dbcd814995f59eec5a23646cae12c08e5ef2e0e9c986198e220d2d038
+    sha256: 944ccc4de05072d2773cfff6b5f279e2058457a672169153ebec0e19f5a3601c
 
 build:
-    number: 3
+    number: 0
     skip: true  # [win]
```
Comment (xylar) [inline recipe/meta.yaml:14]:
```suggestion
    skip: true  # [win]
    script: {{ PYTHON }} -m pip install . --no-deps -vv
```

**Comment by xylar** [comment]
The linter would be happier if you change the license to:
```
license: BSD-3-Clause
```
See the "Short Identifier" here: https://spdx.org/licenses/BSD-3-Clause.html

**Comment by muryanto1** [comment]
@conda-forge-admin, please rerender

**Pull request merged**